docs/source/cluster_management/dropwizard-bundle.rst

```diff
@@ -0,0 +1,31 @@
+.. _dropwizard-bundle:
+
+=================
+Dropwizard bundle
+=================
+
+Overview
+========
+
+The AtlasDB Dropwizard bundle can be added to Dropwizard applications. This will add startup commands to launch
+the AtlasDB :ref:`Console <console>` and :ref:`CLIs <clis>` such as ``sweep`` and ``timestamp``,
+which can be used to perform :ref:`live backups <backup-restore>`.
```

Comment:
This is what the release notes say: ``This will add startup commands to launch the AtlasDB console and CLIs suchs as sweep and timestamp, which is needed to perform live backups.`` Should we change that too?